plugins/poll/lib/polls_validator.rb

```diff
@@ -96,5 +122,45 @@ def valid_multiple_choice_settings?(poll)
 
       true
     end
+
+    def valid_numbers?(poll)
+      return true if poll["type"] != "number"
+
+      valid = true
+
+      min = (poll["min"].presence || 1).to_i
```

Comment:
Here, we should allow for `min >= 0`.

You can also shorten that to

```suggestion
      min = poll["min"].to_i
```

Since `nil.to_i == 0`